### DISCUSSION TITLE
Is it possible to combine a custom lovelace card with a custom integration?

### DISCUSSION THREAD

**Discussion by io-debug** [marq24/ha-senec-v3#79] [Q&A]
Hi,
some days ago I was keen to understand how custom lovelace cards for Home Assistant are build.
Therefore I build a custom card, that could be interesting for users of this SENEC Integration. ([Link to the custom Card](https://github.com/io-debug/io-senec-card))

To use the custom Card a user now has to install the custom SENEC Integration and in addition the custom lovelace card.
Because of this I wonder, if its possible to add the custom card to this SENEC Integration.
My aim: A user just has to install the integration and can use the integration as well as the custom card. 

Happy for any hints and ideas!

**Comment by mschmiedel** [comment]
I am not sure if this is possible so easily, as custom integrations are installed under custom_components and custom UI elements land in the www path. This is true at least for people who manually setup their instances, don't know if other methods allow this. However I will test your card and see if it suits us, seems like a nice way of showing the data... :)
